### ISSUE TITLE
How to invoke Sigmoid Operator in ONNX

### ISSUE THREAD

**Issue by stonelazy** [onnx/onnx#3835]
# Ask a Question
I would like to know how to do i invoke Activation function operators such as [Sigmoid](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sigmoid) or [Relu](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Relu).  

### Question
Am using PytorchLightning and my model code is something like this 
model.py
```
class BoringModel(LightningModule):
    def __init__(self):
        super().__init__()
        self.layer = torch.nn.Linear(32, 2)

    def forward(self, x):
        return self.layer(x)

```
Now, In my test class
```
import torch.nn.functional as F
if __name__ == "__main__":
    model = BoringModel()
    model_out = model(input)
    probs = F.sigmoid(model_out)
    sys.exit(0)
```
When i do ONNX export, i would be exporting BoringModel directly. Later on when am doing inference on the exported ONNX model, am again ending up with torch's dependency (because of F.sigmoid).  
Due to few maintenance reasons, i would not be able to include `F.sigmoid` within my model layer and so its externalized. Having the same above model code, i would like to ONNX prediction and use ONNX's sigmoid operator. Am not sure how to do it.

### Further information
- Relevant Area (inference, test, operators): 

- Is this issue related to a specific model?  
No

**Comment by jcwchen** [comment]
Hi @stonelazy,
Are you saying that you don't want to add Sigmoid op in PyTorch-level and you want to manually add it in the converted ONNX model?

If so, yes it should be doable and you can insert a Sigmoid node into `your_model.graph.node`. For example, https://github.com/onnx/onnx/blob/57edfd7852cec4b9474af8c2ea398c370cc13a79/onnx/examples/add_nodes_to_onnx_model.ipynb (thanks the contributor @Sarthak-Agrawal). Please note that you have to make sure the graph.node is topological-sorted and probably need to change the output shape as well.

**Comment by stonelazy** [comment]
Thanks @jcwchen for your time and response. Yes, partially right. Let me explain it this way.   
I don't want add a SigmoidOp to the converted ONNX model but rather make use of Sigmoid operator of ONNX separately. 
Ex: 
```
output = self.onnx_model(input)
probs = onnx.helper.sigmoid(output)
```
Is this possible ?

**Comment by jcwchen** [comment]
Thanks for the clarification. Take ONNXRuntime as an example. Your probably can achieve it by running two different ONNX models: one is your original main model; the other is a model simply has Sigmoid node.
```
import onnxruntime
main_session = onnxruntime.InferenceSession('onnx_model.onnx')
sigmoid_session = onnxruntime.InferenceSession('sigmoid.onnx')
output = main_session.run([], {'your_input': np.array(data)})[0]
probs = sigmoid_session.run([], {'X': output})
```
Does this meet your requirement?

**Comment by stonelazy** [comment]
Yes absolutely. It does meet my requirement.   
However, i was wondering if we have any magic Python ONNX API that that i can just invoke like `onnx.sigmoid(input)` similar to `torch.nn.functional.sigmoid(model_out)`

**Comment by jcwchen** [comment]
Unfortunately no... You need to either manually add a Sigmoid node after the original ONNX model or run two ONNX models separately during Runtime.
